static/js/containers/HomePage.js

```diff
@@ -94,6 +94,8 @@ const mapStateToProps = state => {
   }
 }
 
-export default R.compose(connect(mapStateToProps), withNavSidebar, withLoading)(
-  HomePage
-)
+export default R.compose(
+  connect(mapStateToProps),
+  withOnlyNavSidebar,
```

Comment:
suggestion you can take or leave: to me it makes a little more semantic sense to have `withNavSidebar` and `withNavAndChannelSidebars` or similar, those names seem more accurate to me